src/modules/network.cpp

```diff
@@ -16,6 +81,21 @@ waybar::modules::Network::Network(const std::string &id, const Json::Value &conf
   if (!id.empty()) {
     label_.get_style_context()->add_class(id);
   }
+
+  auto down_octets = read_netstat(BANDWIDTH_CATEGORY, BANDWIDTH_DOWN_TOTAL_KEY);
```

Comment:
Why this part is needed?
The update method is also called in constructor :)

Reply:
Because InOctets and OutOctets are equal to the total number of octets sent/received since the start of the computer, and as such might not be 0 at the start of Waybar

Otherwise the module may display an artificial network activity spike for the first update